doc/specifications/rest/REST-API-V2.rst

```diff
@@ -210,6 +210,242 @@ SSL Client Authentication
 
 The REST API provides authenticating a user by a subject in a client certificate delivered by the web server configured as SSL endpoint.
 
+Bookmark
+========
+
+Overview
+--------
+
+====================== ================ ================ =============================== ================ ================
+Resource                     POST             GET              HEAD                      PATCH/PUT        DELETE
+---------------------- ---------------- ---------------- ------------------------------- ---------------- ----------------
+/bookmark              .                List bookmarks   .                               .                .
+/bookmark/<path>       Create bookmark  .                Check if location is bookmarked .                Delete bookmark
+====================== ================ ================ =============================== ================ ================
+
+Specification
+-------------
+
+List bookmarks
+~~~~~~~~~~~~~~
+
+:Resource: /bookmark
+:Method: GET
+:Description: List bookmarked locations
+:Parameters:
+    :offset: the offset of the result set
+    :limit: the number of bookmarks returned
+:Headers:
+    :Accept:
+        :application/vnd.ez.api.BookmarkList+xml:  if set the list is returned in XML format
+        :application/vnd.ez.api.BookmarkList+json: if set the list is returned in JSON format
+:Response:
+
+.. code:: http
+
+    HTTP/1.1 200 OK
+    Location: /bookmark
+    Accept-Patch:  application/vnd.ez.api.BookmarkList+(json|xml)
+    ETag: "<newEtag>"
+    Content-Type: <depending on accept header>
+    Content-Length: <length>
+
+:Error Codes:
+        :400: If the Input does not match the input schema definition, In this case the response contains an ErrorMessage_
```

Comment:
What input ? offset and limit ? Does it fail when does are invalid ? Since you're using `getInt()`, I'm not sure it may fail.